app/services/generate_new_week_assignments.rb

```diff
@@ -0,0 +1,41 @@
+class GenerateNewWeekAssignments
+  def initialize(week)
+    @week = week
+  end
+
+  def call
+    generate_last_assignment_list
+
+    2.times do
+      create_week_assignment
+    end
+  end
+
+  private
+
+    def generate_last_assignment_list
+      @last_assignment_list = []
+
+      Employee.all.each do |employee|
+        last_employee_assignment = Assignment.where(employee_id: employee.id).order(:week_id).last
+
+        if last_employee_assignment.present?
+          @last_assignment_list.push(last_employee_assignment)
+        else
+          #new employee who haven't been assigned yet dealt with here
```

Comment:
I guess this is a placeholder/TODO item for later? Is it in Trello?

Reply:
I'm not sure how I'm going to do this yet, so I left it blank for now.